lib/query-interface.js

```diff
@@ -95,7 +95,8 @@ QueryInterface.prototype.createTable = function(tableName, attributes, options,
       if (attributes[keys[i]].type instanceof DataTypes.ENUM) {
         sql = self.QueryGenerator.pgListEnums(tableName, attributes[keys[i]].field || keys[i], options);
         promises.push(self.sequelize.query(
-          sql, { plain: true, raw: true, type: QueryTypes.SELECT, logging: options.logging, transaction: options.transaction }
+          sql,
+          _.assign({ plain: true, raw: true, type: QueryTypes.SELECT }, options)
```

Comment:
might want to change to `_.assign({}, options, { plain: true, raw: true, type: QueryTypes.SELECT })` for good measure